server/views/comAssignedLicencesForHandoverReport.pug

```diff
@@ -0,0 +1,20 @@
+extends layout
```

Comment:
you could create a `reports` folder inside `views/` and move all three reports in there. 

The `/` of view is for the very generic stuff which is shared across everything

Reply:
Maybe admin reports should be in a sub `reports` folder of the `admin` folder 🤔

Reply:
yes sounds good